ci/script/update_rubygems_and_install_bundler

```diff
@@ -2,11 +2,5 @@
 set -e
 source script/functions.sh
 
-if is_ruby_23_plus; then
-  yes | gem update --system
-  yes | gem install bundler
-else
-  echo "Warning installing older versions of Rubygems / Bundler"
-  gem update --system '2.7.8'
-  gem install bundler -v '1.17.3'
-fi
+yes | gem update --system
+yes | gem install bundler
```

Comment:
Can we revert all these changes for now, it makes diffing harder and its easy to clean up post release.